src/libaktualizr/storage/CMakeLists.txt

```diff
@@ -1,3 +1,14 @@
+add_custom_command(OUTPUT sql_schemas.cc
+    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/embed_schemas.py ${PROJECT_SOURCE_DIR}/config/sql/ sql_schemas.cc
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    DEPENDS sql_schemas_target
+)
+
+add_custom_command(
+    OUTPUT sql_schemas_target
+    COMMAND cmake -E echo
+)
```

Comment:
This trick is to make cmake run the command in any case, right?

I tried to remove this custom command, adding a dummy target (sql_schemas_target) as an output of the first `add_custom_command` and it seems to work without outputting an empty line.

Reply:
Yes, here is a diff:

```
diff --git a/src/libaktualizr/storage/CMakeLists.txt b/src/libaktualizr/storage/CMakeLists.txt
index 3a46c643..c55b6e45 100644
--- a/src/libaktualizr/storage/CMakeLists.txt
+++ b/src/libaktualizr/storage/CMakeLists.txt
@@ -1,12 +1,6 @@
-add_custom_command(OUTPUT sql_schemas.cc
+add_custom_command(OUTPUT sql_schemas.cc sql_schemas_target
     COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/embed_schemas.py ${PROJECT_SOURCE_DIR}/config/sql/ sql_schemas.cc
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-    DEPENDS sql_schemas_target
-)
-
-add_custom_command(
-    OUTPUT sql_schemas_target
-    COMMAND cmake -E echo
 )

 if(STORAGE_TYPE STREQUAL "filesystem")
```